azure-pipelines.yml

```diff
@@ -26,9 +26,9 @@ jobs:
       yarn eslint
     displayName: 'Run eslint'
   - script: |
-      python3 -m pip install torch==1.2.0 --user
-      python3 -m pip install torchvision==0.4.0 --user
-      python3 -m pip install tensorflow==1.13.1 --user
+      python3 -m pip install torch==1.3.1 --user
```

Comment:
Can we know for sure whether it works on torch 1.4 and torch 1.5?

Reply:
Added a separate pipeline for latest torch and v1.3.1